src/screens/Recieve/ReceiveScreen.tsx

```diff
@@ -51,7 +54,7 @@ function ReceiveScreen({ route }: { route }) {
   useEffect(() => {
     if (amount) {
       const newPaymentURI = WalletUtilities.generatePaymentURI(receivingAddress, {
-        amount: parseInt(amount) / 10e8,
+        amount: satsEnabled ? parseInt(amount) : parseInt(amount) / 10e8,
```

Comment:
Shouldn't it be:
satsEnabled? parseInt(amount) / 10e8: parseInt(amount)

The payment URI contains the amount [in BTC](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki#examples) and not sats.